apps/condo/domains/user/integration/utils/makeExternalAuth.js

```
@@ -0,0 +1,259 @@
+            const exisingUserSearch = { deletedAt: null, userType }
+            if (userProfile.phone) {
+                exisingUserSearch.phone = userProfile.phone
+            } else {
+                exisingUserSearch.email = userProfile.email
+            }
+
+            const existingUser = await User.getOne(context, exisingUserSearch)
+            if (existingUser) {
+                user = existingUser
+            } else {
+                delete exisingUserSearch.deletedAt
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)